### PR TITLE
add events to CMS

### PR DESCRIPTION
**Pull request by jnakaso**
## What

> Add CMS (i.e. Sanity) support to events.

## Why Do

> We want to manage events without creating releases.

### CODE REVIEW

**Comment by jnakaso** [comment]
I also added a dataset for called "development".  The env.local can be updated to include NEXT_PUBLIC_SANITY_DATASET='development'

File: pages/api/EventsService.ts
Comment (mistyb01) [inline pages/api/EventsService.ts:1]: I like how the query and fetch is handled in this file! I wonder if we should handle fetching the cadre team data the same way for consistency.

File: pages/events.tsx
Comment (mistyb01) [inline pages/events.tsx:1]: I recently merged some design changes to events.tsx, might help to get this file up to date before merging the branch.

**Review by mistyb01 — APPROVED** [review]
This works great, thanks for doing this Jeffrey!

File: pages/api/EventsService.ts [hunk 1]
```diff
@@ -2,55 +2,29 @@
...
-    getResourceFile(): OSEvent[] {
-        return localEvents.map(json => new OSEvent(json));
+    async getOSEvents(): Promise<OSEvent[]> {
+        return await sanityClient()?.fetch(this.eventsQuery)
     }
 
+    // getResourceFile(): async OSEvent[] {
+    //     return [] //localEvents.map(json => new OSEvent(json));
+    // }
+
```
Comment (mistyb01) [inline pages/api/EventsService.ts:25]: remember to remove the comments!

File: pages/api/EventsService.ts
Comment (jnakaso) [reply pages/api/EventsService.ts:1]: I think so, it reduces coupling between modules.  Having said that, it not a high priority.

File: pages/api/EventsService.ts (hunk 1 above)
Comment (jnakaso) [reply pages/api/EventsService.ts]: thx, done

**Pull request merged**